client/utils/isProd.js

```diff
@@ -0,0 +1,9 @@
+let isPubPubProduction;
+
+export const setIsProd = (val) => {
```

Comment:
This doesn't seem to be called anywhere. Previously, we set isPubPubProduction within hydrateWrapper right after we parsed the initialData.

Reply:
Oh, great catch.